packages/field/src/InteractionStateMixin.js

```diff
@@ -16,7 +16,7 @@ import { FocusMixin } from './FocusMixin.js';
 export const InteractionStateMixin = dedupeMixin(
   superclass =>
     // eslint-disable-next-line no-unused-vars, no-shadow
-    class InteractionStateMixin extends CssClassMixin(FocusMixin(ObserverMixin(superclass))) {
+    class InteractionStateMixin extends ObserverMixin(superclass) {
       static get properties() {
         return {
```

Comment:
removes the CssClassMixin as it "breaks" the prototype chain...

should use `reflect: true` for most properties and should for backward compatibility still add the css classes

Reply:
MR is here: https://github.com/ing-bank/lion/pull/159